apps/passport-client/new-components/Input.tsx

```diff
@@ -0,0 +1,42 @@
+import { InputHTMLAttributes, Ref } from "react";
+import styled from "styled-components";
+
+export const BigInput = styled.input`
```

Comment:
for reimplementations of existing components, i would prefer their names to not overlap with the existing components, so as to not hinder development on existing UIs. Something like `BigInput2` would work for me